requirements/env_omnisci.yml

```diff
@@ -3,7 +3,7 @@ channels:
   - conda-forge
 dependencies:
   - pandas==1.3.5
-  - pyarrow==5.0.0
+  - pyarrow==6.*
```

Comment:
```suggestion
  - pyarrow=6
```
or `pyarrow 6.*` - no point of mixing asterisk and equality

Reply:
Why do we want to use a fuzzy constraint for pyarrow? OmniSci works with an exact pyarrow version, doesn't it?

Reply:
Omnisci recipe has "set arrow_version = "6.*"" so I think this is fine

Reply:
fixed